src/qrcode/utils.ts

```diff
@@ -112,6 +112,9 @@ export function* handleBarcode(
   requesterAddress?: string
 ) {
   const walletConnectEnabled: boolean = yield call(isWalletConnectEnabled, barcode.data)
+  if (/^0x[a-f0-9]{40}$/gi.test(barcode.data)) {
```

Comment:
would be cool to leave a comment to make life easier for regex noobs like me (or put this in a function because we use it in the QRScanner component too) 🙏🏻

Reply:
Now that this is done at this level, we can remove the special handling in https://github.com/valora-inc/wallet/blob/3f701255858a783abadaff4045d137367a69eb8c/src/qrcode/QRScanner.tsx#L62-L65

Reply:
sounds good! I wasn't sure if the two builds have two separate paths. Will remove this.